Bottom: aae0fa426463446189006203d2abbfe27e82a7b5
Top:    029799627a9af223a6c5194a8fbfeaf12faed605
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-09-05 12:53:53 +0100

hid/gtk: Turn on defines to debug a potential GTK-3.0 build


---

diff --git a/configure.ac b/configure.ac
index 71ab1c4..d2e2eeb 100644
--- a/configure.ac
+++ b/configure.ac
@@ -770,6 +770,9 @@ $GTKGLEXT_PKG_ERRORS])]
 	GTKGLEXT_VER=`$PKG_CONFIG gtkglext-1.0 --modversion`
 	fi
 
+	GTK_CFLAGS="$GTK_CFLAGS -DGSEAL_ENABLE"
+#	GTK_CFLAGS="$GTK_CFLAGS -DGDK_DISABLE_DEPRECATED -DGTK_DISABLE_DEPRECATED -DGSEAL_ENABLE"
+
 	;;
 
       gcode|nelma|png )
